server/api/cart.js

```diff
@@ -40,3 +40,17 @@ router.put('/add', async (req, res, next) => {
     next(error)
   }
 })
+
+router.delete('/:id', async (req, res, next) => {
+  try {
+    if (req.user) {
+      const cart = await User.findByPk(req.body.cartId)
+      const mug = await Mug.findByPk(req.body.mugId)
+      cart.delete(mug)
+      res.json(cart)
+    }
+  } catch (error) {
+    console.log('Error in the delete item from cart route', error)
+    next(error)
```

Comment:
couple things I think should be different with this:
--the URI is to /api/cart/:id but the req.params.id is never used. Maybe should just be router.delete('/')?
--to pull the cart we should be invoking findByPk on the Order model not User, and also we can access the cartId right from req.user so we don't have to send it in the req.body, so line 47 could be:
   const cart = await Order.findByPk(req.user.cartId) (same way we pull the cart in get cart route/method)
--isn't cart.delete() the sequelize method that would just remove the cart from the database? I think what we want to take the mug off the cart is this magic method: cart.removeMug(mug)
--Not sure what is most useful to send back to the reducer at the end, maybe res.json(req.body.mugId)?